gravitee-apim-console-webui/src/management/application/application-navigation/application-navigation.component.ts

```diff
@@ -129,6 +122,10 @@ export class ApplicationNavigationComponent implements OnInit, OnDestroy {
     ]);
 
     this.selectedItemWithTabs = this.subMenuItems.find((item) => item.tabs && this.isTabActive(item.tabs));
+
+    this.router.events.subscribe(() => {
```

Comment:
as there is a subscription, you have to manage the unsubscribable with `takeUntil(this.unsubscribe$),`

we can also improve it to merge with previous  set of selectedItemWithTabs : 

```
this.router.events.pipe(
startWith({}),
takeUntil(this.unsubscribe$),
).subscribe(() => {...

```

Reply:
Done